src/eolab/rastertools/processing/sliding.py

```diff
@@ -56,8 +56,12 @@ def compute_sliding(input_image: str, output_image: str, rasterprocessing: Raste
             if src.height < blockysize:
                 blockysize = utils.highest_power_of_2(src.height)
 
-            # dtype of output data
+            # dtype and creation options of output data
             dtype = rasterprocessing.dtype or rasterio.float32
+            processing_dtype = rasterprocessing.processing_dtype or dtype
+            nbits = rasterprocessing.nbits
+            compress = rasterprocessing.compress
```

Comment:
```suggestion
            compress = rasterprocessing.compress or profile.compress or 'lzw'
```

Not sure about the `profile.compress`, but the idea is to either set to the specified one, keep the src.profile compression if it exist, or force it to lzw, compression is always better!

Reply:
Ok I get the idea. It would be 'src.compression' then, 'profile.compress' or 'profile.compression' does not work